src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheck.java

```diff
@@ -66,6 +78,14 @@
  * r = 5; int t; //violation here
  * </pre>
  * <p>
+ * An example of check with enabled processing resources
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;OneStatementPerLine&quot;&gt;
+ * &lt;property name=&quot;treatTryResourcesAsStatement&quot; value=&quot;true&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
  * Note: resource declarations can contain variable definitions
  * and variable references (from java9).
  * This check is only applied to variable definitions.
```

Comment:
It seems to me this `Note` has to be updated.

Reply:
updated, please take a look

Reply:
The note was updated, but the example after it makes the update confusing since it says when `"treatTryResourcesAsStatement" is enabled` yet example and further wording goes on when it isn't enabled.

I think the new wording should talk about when `treatTryResourcesAsStatement` is off, and mention if you disagree with this you can turn the property on.

Reply:
> example and further wording goes on when it isn't enabled.

No, example shows violation in resources statements, it is possible if property is enabled.

> I think the new wording should talk about when `treatTryResourcesAsStatement` is off, and mention if you disagree with this you can turn the property on

I disagree. When property is enabled, it skips variable references (https://github.com/checkstyle/checkstyle/issues/6125) and it should be mentioned in docs.